packages/app/src/app/components/Create/CreateBox/CreateBoxForm.tsx

```diff
@@ -124,6 +129,37 @@ export const CreateBoxForm: React.FC<CreateBoxFormProps> = ({
           />
         </Stack>
 
+        <Stack direction="vertical" gap={1}>
+          <Text size={3} as="label">
+            Folder
+          </Text>
+
+          <Query
```

Comment:
didn't know we have this :D

Reply:
I found it in another component I remembered (the sandbox move modal) where we show all collections 😄